CHANGELOG.md

```diff
@@ -13,6 +13,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
+- Unclear error message when indexing a `nil` value (#33).
```

Comment:
I think we should talk about user experience here.

```suggestion
- Unclear error message when `roles.httpd` config is not yet applied (#33).
```

Reply:
Yes, edited, thanks.